robots/cmd/perf-report-creator/output/vm/CREATE-pods-count/2022-10-10/data/results.json

```diff
@@ -0,0 +1,17 @@
+{
+  "StartDate": "2022-10-10",
+  "NumberOfDays": 0,
```

Comment:
Is this an example of real data you scraped?

Reply:
no @rthallisey these are made up numbers to demo the output directory structure.